src/patternfly/patternfly-charts.scss

```diff
@@ -293,10 +293,10 @@ $chart: #{$pf-prefix} + 'chart';
   --#{$chart}-threshold--stroke--Width: 1.5;
 
   // Tooltip
-  --#{$chart}-tooltip--corner-radius: 0;
+  --#{$chart}-tooltip--corner-radius: 6;
```

Comment:
Not sure if you have a good way to do this on the react side, but I wonder if you could use the `--pf-t--global--border--radius--small` token here (and for `--#{$chart}-tooltip--flyoutStyle--corner-radius` below), and strip the unit in react-charts when you reference `chart_tooltip_corner_radius.value` and `chart_tooltip_flyoutStyle_corner_radius.value`?

FWIW, here's the tooltip style that references that same token
https://github.com/patternfly/patternfly/blob/b93029abee9117da890c05f219412e3c423d57b4/src/patternfly/components/Tooltip/tooltip.scss#L16

Reply:
I don't think it's a good idea to include units here -- Victory cannot use them. This would unnecessarily introduce a mix of numbers and strings in react-tokens. Not only would be forced to later strip units, but anyone using a custom tooltip would need to do the same.